app/api/stripe/manage-subscription/route.ts

```
@@ -0,0 +1,178 @@
+          message: 'Subscription will be canceled at the end of the current period',
+          cancelAtPeriodEnd: true
+        });
+
+      case 'upgrade':
+      case 'downgrade':
+        if (!subscription?.stripeSubscriptionId) {
+          return NextResponse.json({ error: 'No active subscription' }, { status: 400 });
+        }
+
+        // Get the new plan price ID from your SUBSCRIPTION_PLANS
+        const { SUBSCRIPTION_PLANS } = await import('@/lib/stripe');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)